splunklib/searchcommands/generating_command.py

```diff
@@ -56,7 +56,7 @@ class GeneratingCommand(SearchCommand):
     +==========+=====================================+============================================+
     | streams  | streaming=True[,local=[True|False]] | type='streaming'[,distributed=[true|false] |
     +----------+-------------------------------------+--------------------------------------------+
-    | events   | retainsevents=True, streaming=False | type='eventing'                            |
+    | events   | retainsevents=True, streaming=False | type='events'                              |
```

Comment:
Nope..not a type..we were actually expecting 'eventing', whereas in splunk code we were expecting 'events'.